enzy_htp/preparation/protonate.py

```diff
@@ -304,13 +310,41 @@ def _fix_pybel_output(pdb_path: str, out_path: str, ref_name_path: str = None) -
     use {ref_name_path} to maximumly keep original atom names and residue name (if provided)
     * this fix is based on all newly added atoms from pybel will be under original ones
     * i.e.: assume the original atoms is a subset of the final atoms and in its original order
+
+    Args:
+        pdb_path: Protonated PDB filepath with messed up atom names and residue names.
+        out_path: Fixed PDB filepath.
+        ref_name_path: Reference PDB filepath for atom names and residue name.
+
+    Returns:
+        None.
     """
     if ref_name_path is not None:
```

Comment:
You should use the `PDBParser` if at all possible.

Reply:
Yeah, this is a design that I hope _fix_pybel_output independent from Structure() at the time. But I guess it don't much that much sense for now. I say we can keep it for now. (we will use pybel less and less I think)